### PR TITLE
movie app, but without form adding a new movie :(

### PR DESCRIPTION
**Pull request by jbd3**

### CODE REVIEW

File: JakeDouglas-jbd3/movie_app/config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  #   resources :posts, concerns: :toggleable
+  #   resources :photos, concerns: :toggleable
+
+  # Example resource route within a namespace:
+  #   namespace :admin do
+  #     # Directs /admin/products/* to Admin::ProductsController
+  #     # (app/controllers/admin/products_controller.rb)
+  #     resources :products
+  #   end
+end
```
Comment (Bkahler) [inline JakeDouglas-jbd3/movie_app/config/routes.rb]: Delete this end

File: JakeDouglas-jbd3/movie_app/config/routes.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
+Rails.application.routes.draw do
+  root "home#index"
```
Comment (Bkahler) [inline JakeDouglas-jbd3/movie_app/config/routes.rb]: here you need to change "home" to "movies" the first part of the code should refer to the controller and the part after the # should refer to the method.

**Comment by Bkahler** [comment]
Hey,
I started going through the code for the movies app, and it looks like there was a lot of copy pasting going on. Your routes are the root (no pun intended) of all your issues. You copied the urls routes into the movies app. Please avoid copy/pasting more than a line or so at a time. When you are just starting out, its good to type things fresh. This will prevent your from making sweeping mistakes. Please fix the routes and resubmit when you can.

**Comment by jbd3** [comment]
Ok, I fixed that, but now is with the forms...



Jake Douglas
Community Manager
GiftRocket

**Pull request merged**